facdb/sql/usdot_airports.sql

```diff
@@ -2,30 +2,31 @@ DROP TABLE IF EXISTS _usdot_airports;
 SELECT
     uid,
     source,
-    fac_name as facname,
-    NULL as addressnum,
-    NULL as streetname,
-    NULL as address,
+    name as facname,
+    parsed_hnum as addressnum,
+    parsed_sname as streetname,
+    cleaned_address as address,
     city,
-    NULL as zipcode,
+    zipcode as zipcode,
     county as boro,
     NULL as borocode,
     NULL as bin,
     NULL as bbl,
-    fac_type as factype,
+    facility_type as factype,
     'Airports and Heliports' as facsubgrp,
     (CASE
-        WHEN owner_type = 'Pr' THEN fac_name
+        WHEN ownership= 'PR' THEN name
         ELSE 'Public'
     END) as opname,
     (CASE
-        WHEN owner_type = 'Pr' THEN 'Non-public'
+        WHEN ownership = 'PR' THEN 'Non-public'
         ELSE 'Public'
     END) as opabbrev,
     'USDOT' as overabbrev,
     NULL as capacity,
     NULL as captype,
-    wkt::geometry as wkb_geometry,
+    --- wkt::geometry as wkb_geometry,
+    NULL as wkb_geometry,
```

Comment:
I am not entirely sure if we want the wkb_geometry to be null

Reply:
This is a great question Max, this also gave me pause. The way I understand the facDB to work is that wkb_geometry is populated later on in the pipeline. I came to this conclusion after looking at other datasources that also select wkb_geometry as NULL in this step. For example `dep_wwtc` also selects wkb_geometry as NULL in it's associated .sql file, but some (actually all but the exact number doesn't matter) of the facilites are mapped in the facdb table on my machine.
I don't know why it's built this way and I don't think it makes sense for some datasources to populate wkb_geometry here and others not to. But that's how it is written and so this should be a non-breaking change

Reply:
If there is no geometry from the source data wkb_geometry is set to NULL in this step.  See other examples [here](https://github.com/NYCPlanning/db-facilities/search?q=%22NULL+as+wkb_geometry%22).  The spatial data is then assigned in [this step](https://github.com/NYCPlanning/db-facilities/blob/58d104978ad6757d66da4cf16964e54a16d3a855/facdb/sql/_create_facdb_geom.sql#L21) and it gets the spatial data from 1b.